bftengine/src/bftengine/ReplicaForStateTransfer.cpp

```diff
@@ -44,6 +45,7 @@ ReplicaForStateTransfer::ReplicaForStateTransfer(const ReplicaConfig &config,
 void ReplicaForStateTransfer::start() {
   stateTranTimer_ = TimersSingleton::getInstance().add(
       5s, Timers::Timer::RECURRING, [this](Timers::Handle h) { stateTransfer->onTimer(); });
+  metric_state_transfer_timer_.Get().Set(5000);
```

Comment:
Please declare a variable for this value.

Reply:
done